src/App.js

```diff
@@ -1,5 +1,34 @@
+import { Routes, Route } from "react-router-dom";
+import styled from "styled-components";
+
+import Navigation from "./components/Navigation";
+import AboutUsPage from "./pages/AboutUsPage";
+import HomePage from "./pages/HomePage";
+import PicturePage from "./pages/PicturePage";
+
 function App() {
-  return <div className="App">Test</div>;
+  return (
+    <StyledAppGrid>
+      <Navigation />
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/picture" element={<PicturePage />} />
+        <Route path="/aboutUs" element={<AboutUsPage />} />
+      </Routes>
+      <StyledFooter aria-label="footer">copyright by xyz</StyledFooter>
+    </StyledAppGrid>
+  );
 }
 
 export default App;
+
+const StyledAppGrid = styled.div`
+  height: 100vh;
+  display: grid;
+  grid-template-rows: 48px 1fr 48px;
+  justify-items: center;
+`;
+
+const StyledFooter = styled.footer`
+  align-self: end;
+`;
```

Comment:
This sorta thing I'd recommend putting into a .css file and then importing it from there where you need it. remember to keep things tidy and concise! There's always something that can be cleaned up and made easier to read. They will be looking for that!